packages/sacado/src/Kokkos_DynRankView_Fad.hpp

```diff
@@ -1246,7 +1255,11 @@ create_mirror( const Kokkos::DynRankView<T,P...> & src
 }
 
 template<class Space, class T, class ... P>
+#if KOKKOS_VERSION >= 40499
+typename Impl::MirrorDRViewType<Space,T,P ...>::dest_view_type
+#else
 typename Impl::MirrorDRVType<Space,T,P ...>::view_type
```

Comment:
I know this needed to be updated to `MirrorDRViewType` , but I'm not certain that `dest_view_type` is the correct return type, instead of `view_type`?

Reply:
I think it is but deferring to @